### DISCUSSION TITLE
Intelliflo VSF pump settings

### DISCUSSION THREAD

**Discussion by mhmatttu** [tagyoureit/nodejs-poolController#609] [App and O/S Config Questions] [answered]
I setup njsPC for my brother-in-law on his EasyTouch2 8 panel.  Pump settings were all reporting in RPM vs GPM.  When attempting to delete a pump circuit via DashPanel (which will not delete), all settings have changed to GPM.  attempts to change back to RPM in DashPanel will not save.  Changes to the GPMs in the settings menu will not save either.  

I can change the GPM from the main screen and will save in the settings screen but not vice versa.  

I need to change back to RPM and re-set the RPM speeds

![image](https://user-images.githubusercontent.com/47801991/190267262-70d17046-88c5-445a-aded-9f75c11219b9.png)

![image](https://user-images.githubusercontent.com/47801991/190267359-5b06a9fc-77b5-4f63-83b6-5573c985050a.png)

I am remote so not able to access the ET panel.

Any thoughts on this?  Thanks!

**Comment by rstrouse** [comment]
Are you getting an error message when you try to delete the circuit?

**Reply by rstrouse** [reply]
Is there any way I can get a replay while you are trying to delete the circuit?

**Reply by mhmatttu** [reply]
No error messages.  When I delete the circuit, it goes away but when I click save pump, it returns.  Attached is a replay. 

[replay (3).zip](https://github.com/tagyoureit/nodejs-poolController/files/9570266/replay.3.zip)

**Comment by rstrouse** [comment, marked as answer]
Pull njsPC it should allow circuits to be deleted off the pump.

**Reply by mhmatttu** [reply]
Perfect.  I pulled njsPC and was able to successfully delete a pump circuit.  However when I change the pump circuit settings from GPM to RPM and adjust to the desired RPMs, they do not save.  They revert back to gpm.  

Attached is a replay of me adjusting each circuit to RPM and then adjusting the RPM speed on all 4 remaining circuits and then clicking save pump.  As noted above, after I go back to the home screen and click on the pump, the circuits have reverted back to GPM and when I go back to the pump settings menu, each circuit has reverted back to GPM.  

I appreciate the help.  

[replay (4).zip](https://github.com/tagyoureit/nodejs-poolController/files/9571811/replay.4.zip)

**Reply by rstrouse** [reply]
Pull njsPC and dashPanel.  I don't have a good example of somebody setting the circuits on a VSF pump in ET or ScreenLogic but I think I have the correct units byte.  However, this may be inverted because I have not witnessed a replay where somebody is changing the units via ET or ScreenLogic.  It appears all the VSF pump replays I have are for IntelliCenter.

**Reply by mhmatttu** [reply]
That fixed it.  I continue to be amazed!  Thank you!!
